APITestingFramework/Features/User/Put.feature

```diff
@@ -1,14 +1,14 @@
-Feature: User Update in https://todo.ly/ website. API endpoint https://todo.ly/api/user.xml
+Feature: User Update
     As an authenticated user, I want to be able to update my user account.
 
     Scenario: Update a user
-        Given the user is authenticated and has permission to update a user
+        Given the user is authenticated
         When the user submits a PUT request to the API endpoint with a valid JSON or XML payload
         Then the API should return a 200 status code and the user should be updated in the database
 
     Scenario: Update a user with invalid user credentials
-        Given the user is authenticated and has permission to update a user
-        When the user submits a PUT request to the API endpoint with an invalid user ID in the URL
+        Given the user is authenticated
+        When the user submits a PUT request to the API endpoint with an invalid user email
```

Comment:
it's better put the example int the step i.e.

the user submits a PUT request to the API endpoint with "invalid@correo" invalid user email

Reply:
Done